src/main/java/mate/academy/hibernate/relations/lib/Injector.java

```diff
@@ -0,0 +1,174 @@
+package mate.academy.hibernate.relations.lib;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Injector {
```

Comment:
why do we need injector here? Lets remove it. Same for Dao, Inject Ans Service annotation